lua/lvim/core/telescope.lua

```diff
@@ -153,4 +153,15 @@ function M.setup()
   end
 end
 
+-- Smartly opens either git_files or find_files, depending on whether the working directory is
+-- contained in a Git repo.
+function M.git_files_with_fallback()
+  local builtin = require "telescope.builtin"
+  local ok = pcall(builtin.git_files)
+
+  if not ok then
+    builtin.find_files()
+  end
+end
+
```

Comment:
Please move this to `telescope/custom-finders` and then we can merge it, since it's a good addition anyway.

Reply:
Maybe call it `find_project_files()` instead?

Reply:
Sure, can do. 🙂